src/parser/classes/livechat/items/LiveChatSponsorshipsHeader.ts

```diff
@@ -0,0 +1,39 @@
+import { Parser } from '../../../index.js';
+import { YTNode } from '../../../helpers.js';
+import type { ObservedArray } from '../../../helpers.js';
+import type { RawNode } from '../../../index.js';
+import NavigationEndpoint from '../../NavigationEndpoint.js';
+import Text from '../../misc/Text.js';
+import Thumbnail from '../../misc/Thumbnail.js';
+import LiveChatAuthorBadge from '../../LiveChatAuthorBadge.js';
+
+export default class LiveChatSponsorshipsHeader extends YTNode {
+  static type = 'LiveChatSponsorshipsHeader';
+
+  author_name: Text;
+  author_photo: Thumbnail[];
+  primary_text: Text;
+  author_badges: ObservedArray<LiveChatAuthorBadge> | null;
+  menu_endpoint: NavigationEndpoint;
+  context_menu_accessibility: {
+    accessibility_data: {
+      label: string
+    }
+  };
```

Comment:
As this object only contains a single nested field it might make sense to flatten it so that the string is a top-level field. Name idea: `context_menu_accessibility_label`

Reply:
Done!